File: Python/01.ProgrammingBasics/06.Conditional-Statements-Adv-Exercise/01.Cinema/_01.Cinema.py
projection_type = input()
rows = int(input())
columns = int(input())

ticket_price = 0

if projection_type == "Premiere":
    ticket_price = 12.00
elif projection_type == "Normal":
    ticket_price = 7.50
elif projection_type == "Discount":
    ticket_price = 5.00

total_income = rows * columns * ticket_price

print(f"{total_income:.2f} leva")

File: C++/01-Basics/03.3-ConditionalStatementsAdvЕxercise/01.Cinema/01.Cinema.cpp
#include <iostream>
#include <iomanip>
#include <string>

using namespace std;

int main() {
    const double PREMIERE_PRICE = 12.00;
    const double NORMAL_PRICE = 7.50;
    const double DISCOUNT_PRICE = 5.00;

    string projectionType;
    int rows, columns;
    cin >> projectionType >> rows >> columns;

    double totalIncome = 0.0;

    if (projectionType == "Premiere") {
        totalIncome = PREMIERE_PRICE * rows * columns;
    }
    else if (projectionType == "Normal") {
        totalIncome = NORMAL_PRICE * rows * columns;
    }
    else if (projectionType == "Discount") {
        totalIncome = DISCOUNT_PRICE * rows * columns;
    }

    cout << fixed << setprecision(2) << totalIncome << " leva" << endl;

    return 0;
}